app/models/html_document.rb

```diff
@@ -49,10 +49,13 @@ def parse_content
   end
 
   def tag_scrubber
-    # convert custom tags to plain 'ol divs to ensure they are not
-    # stripped out during HTML sanitization
     Loofah::Scrubber.new do |node|
+      # convert custom tags to plain 'ol divs to ensure they are not
+      # stripped out during HTML sanitization
       node.name = 'div' if /-/.match(node.name)
+
+      # omit common elements
+      node.remove if %w[footer nav].include?(node.name)
     end
```

Comment:
Wouldn't it be easier to read to do  node.name =~ /footer|nav/ ? And you are not looping through an array.

Reply:
I find it easier to read as-is, and I try to avoid using regexes unless it's necessary. In this case, it's not necessary, because we're looking for an exact string match. The performance difference between the two approaches is negligible:
```ruby
[17] Usasearch(main)> Benchmark.realtime{ 1000.times{  %w[nav footer].include? '<nav>' } }
0.005774999997811392
[18] Usasearch(main)> Benchmark.realtime{ 1000.times{  '<nav>' =~ /nav|footer/ } }
0.004207999998470768
```